lib/appsignal/transaction.rb

```diff
@@ -99,6 +99,8 @@ def set_metadata(key, value)
 
     def set_error(error)
       return unless error
+      return if Appsignal.is_ignored_error?(error)
```

Comment:
I think this actually makes the behavior of the new version consistent with the old one. We used to have `Appsignal.add_exception` in the integrations, which also does this.